src/components/ChatLog.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import './ChatLog.css';
+import PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry';
+
+const ChatLog = ({entries, onToggleLike}) => {
+
+    const ChatComponents = entries.map((message) => {
+        return (
+        <div>
+            <ChatEntry 
+            id={message.id} 
+            sender={message.sender} 
+            body={message.body} 
+            timeStamp={message.timeStamp} 
+            liked={message.liked} 
+            onToggleLike={onToggleLike}/>
+        </div>)
```

Comment:
Note that you don't need to wrap `<ChatEntry>` in a `<div>` since you are only returning one element. I'd recommend getting rid of the `<div>` to cut down on the number of HTML elements.